src/components/Router/Router.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Routes, Route } from "react-router-dom";
+import { PageHome } from "../PageHome";
+import { PageMovies } from "../PageMovies";
+import { PageTVShows } from "../PageTVShows";
+import { PagePeople } from "../PagePeople";
+import { PageMore } from "../PageMore";
+import { PageNotFound } from "../PageNotFound";
+
+export const Router = () => (
+  <Routes>
+    <Route path="/" element={<PageHome />} />
+    <Route path="/movies" element={< PageMovies />} />
+    <Route path="/tvshows" element={< PageTVShows />} />
+    <Route path="/people" element={< PagePeople />} />
+    <Route path="/more" element={< PageMore />} />
+    <Route path="*" element={< PageNotFound />} />
```

Comment:
no need to add "Page" before all components. Just use a more convenient name

Reply:
Did this so that they stayed together in the components folder.
Made the subfolder Pages in the components folder, moved them there. Hope that's ok

Reply:
just leave as before, but without 'page'

Reply:
done